pkg/scheduler/framework/plugins/noderesources/least_allocated.go

```diff
@@ -63,3 +71,27 @@ func (la *LeastAllocated) ScoreExtensions() framework.ScoreExtensions {
 func NewLeastAllocated(_ *runtime.Unknown, h framework.FrameworkHandle) (framework.Plugin, error) {
 	return &LeastAllocated{handle: h}, nil
 }
```

Comment:
This will become something like this:
```
return &LeastAllocated{
  handle: h,
  resourceAllocationScorer: {LeastAllocatedName, leastResourceScorer, DefaultRequestedRatioResources},
 }, nil
```